docs/optimization_guide/dldt_deployment_optimization_tput_advanced.md

```diff
@@ -1,77 +1,81 @@
 
-> **NOTE**: Using the [High-Level Performance Hints](../OV_Runtime_UG/performance_hints.md) is the alternative,  portable and future-proof option, allowing the OpenVINO to find best combination of streams and batching for a given scenario and model. 
+> **NOTE**: Using the [High-Level Performance Hints](../OV_Runtime_UG/performance_hints.md) is the alternative, portable and future-proof option, allowing the OpenVINO to find best combination of streams and batching for a given scenario and model. 
 
+@anchor stream_considerations
 ### Number of Streams Considerations
-* Select the number of streams is it is **less or equal** to the number of requests that your application would be able to runs simultaneously
-* To avoid wasting resources, the number of streams should be enough to meet the _average_ parallel slack rather than the peak load
-* As a more portable option (that also respects the underlying hardware configuration) use the `ov::streams::AUTO`
-* It is very important to keep these streams busy, by running as many inference requests as possible (e.g. start the newly-arrived inputs immediately)
-   * Bare minimum of requests to saturate the device can be queried as `ov::optimal_number_of_infer_requests` of the  `ov:Compiled_Model` 
-* _Maximum number of streams_ for the device (per model) can be queried as the `ov::range_for_streams`
+* Select the number of streams that is **less or equal** to the number of requests that application would be able to runs simultaneously.
+* To avoid wasting resources, the number of streams should be enough to meet the _average_ parallel slack rather than the peak load.
+* As a more portable option (that also respects the underlying hardware configuration) it is advised to use the `ov::streams::AUTO`.
+* It is very important to keep these streams busy, by running as many inference requests as possible (e.g., start the newly-arrived inputs immediately):
+   * Bare minimum of requests to saturate the device can be queried as `ov::optimal_number_of_infer_requests` of the  `ov:Compiled_Model`.
+* _Maximum number of streams_ for the device (per model) can be queried as the `ov::range_for_streams`.
 
 ### Batch Size Considerations
-* Select the batch size that is **equal** to the number of requests that your application is able to runs simultaneously
-   * Otherwise (or if the number of "available" requests fluctuates), you may need to keep several instances of the network (reshaped to the different batch size) and select the properly sized instance in the runtime accordingly 
+* Select the batch size that is **equal** to the number of requests that your application is able to runs simultaneously:
+   * Otherwise (or if the number of "available" requests fluctuates), you may need to keep several instances of the network (reshaped to the different batch size) and select the properly sized instance in the runtime accordingly .
```

Comment:
```suggestion
   * Otherwise (or if the number of "available" requests fluctuates), you may need to keep several instances of the network (reshaped to the different batch size) and select the properly sized instance in the runtime accordingly.
```